src/superstructor/re_frame/fetch_fx.cljs

```diff
@@ -117,19 +125,39 @@
    :final-uri?  (.-useFinalURL js-response)
    :headers     (js-headers->clj (.-headers js-response))})
 
-(defn response->reader-kw
-  "Returns a keyword of the type of reader to use for the body of the
-   response according to the Content-Type header."
+(defn ->reader
+  "Wrap a normal keyword on a reader function (see `response->header`), and
+   set a default json body reader."
+  [reader-or-kw]
+  (cond
+    ;; default json body reader
+    (and (keyword? reader-or-kw)
+         (#{:json} reader-or-kw)) default-json-reader
```

Comment:
This is a bit heavy handed. I faced the issue of providing default behaviour for json while at the same time catering for user config overriding it.. Namely regex pattern identity (or inclusion) and map elements order..
I would have liked to do the usual `(merge defaults-map user-config-map)`